util/find-include-nits

```diff
@@ -0,0 +1,79 @@
+#! /usr/bin/env perl
+# Copyright 2021 The OpenSSL Project Authors. All Rights Reserved.
+#
+# Licensed under the Apache License 2.0 (the "License").  You may not use
+# this file except in compliance with the License.  You can obtain a copy
+# in the file LICENSE in the source distribution or at
+# https://www.openssl.org/source/license.html
+
+# Run this program like this:
+#       perl util/find-include-nits [files...]
+#
+# If given no arguments, will open a pipe to 'git ls-files'
+
+use strict;
+use warnings;
+
+my $status = 0;
+
+# Print error message, set $status.
+sub err {
+    print join(" ", @_), "\n";
+    $status = 1
+}
+
+sub
+process
+{
+    my $file = shift;
+
+    # If it's foo.in, reach foo
+    my $infile = $file;
+    $infile =~ s/\.in$//;
+
+    my $IN;
+    if ( !open($IN, '<', $infile) ) {
+        err("Can't open $infile, $!");
+        return;
+    }
```

Comment:
This unfortunately won't work for out-of-tree builds.

Reply:
> This unfortunately won't work for out-of-tree builds.

Yes out-of-tree builds don't work because of `git ls-files`  I could prepend `$SRCDIR/` if you really want.  But other things still break:
```
/usr/bin/perl "-I." -Mconfigdata "../github/util/dofile.pl" "-oMakefile" ../github/providers/common/der/der_digests_gen.c.in > providers/common/der/der_digests_gen.c
Undefined subroutine &oids_to_c::process_leaves called at ../github/providers/common/der/der_digests_gen.c.in line 19.
Makefile:19688: recipe for target 'providers/common/der/der_digests_gen.c' failed
make: *** [providers/common/der/der_digests_gen.c] Error 1
        exit 2
```